lua/gatekeeper/lpmlib.lua

```diff
@@ -20,3 +20,29 @@ function lpm_add_tbl8s(ip_addr, prefix_len, prefixes)
 	return 1
 end
 
+function lpm6_add_tbl8s(ip6_addr, prefix_len, prefixes)
+	local prefix
+	local masked_ip
+	local depth = 24
+	local ret = 0
+
+	-- For a prefix with length longer than 24, one tbl8
+	-- is needed every 8 bits. If the prefix length is not
+	-- a multiple of 8, then prefix expansion will be performed
+	-- on that tbl8 entry. More details can be found on
+	-- the addtion description in DPDK LPM6 library:
+	-- https://doc.dpdk.org/guides/prog_guide/lpm6_lib.html#addition
+	while (depth < prefix_len)
+	do
```

Comment:
Style: drop the parenthesis (Lua does not require them), and move the `do` to the end of the previous line.